docs/getting-started/app-custom-theme.md

```diff
@@ -76,60 +82,67 @@ want to use the default themes, they are exported as `lightTheme` and
 ## Example of a custom theme
 
 ```ts
-const themeOptions = createThemeOptions({
+import {
+  createTheme,
+  genPageTheme,
+  lightTheme,
+  shapes,
+} from '@backstage/theme';
+
+const customTheme = createTheme({
```

Comment:
This was previously `createThemeOptions`, is that fine?

Reply:
Yeah that seems like a more useful example. Could call it `myTheme` to make it line up with the example above